src/containers/CharacterList.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { getCharacters, getCharacter } from '../services/airbenderApi';
+import DeckOfCards from '../components/DeckOfCards/DeckOfCards';
+
+export default class CharacterList extends Component {
+  static propTypes = {
+    history: PropTypes.object.isRequired,
+    match: PropTypes.shape({
+      params: PropTypes.shape({
+        name: PropTypes.string.isRequired
+      }).isRequired
+    }).isRequired
+  }
+
+  state = {
+    count: 10,
+    page: 1,
+    characters: []
+  };
+
+  componentDidMount() {
+    if(this.props.match.params.name) {
+      getCharacter(this.props.match.params.name)
+        .then(json => this.setState({ characters: json }));
+    }
+    else {
```

Comment:
this functionality doesn't seem to be working - if I don't search for a character, I get a "Not Found" message when I go to the list page